overrides/home.html

```diff
@@ -11,6 +11,7 @@
     <link rel="stylesheet" href="./assets/landing/spotlight.css">
     <link rel="stylesheet" href="./assets/landing/socials.css">
     <script src="./assets/landing/home.js"></script>
+    <script src="./assets/javascript/redirections.js"></script>
```

Comment:
For adding javascript that is suppose to be global it's better to add
```
extra_javascript:
  - assets/javascript/redirections.js
  ``` 
to the mkdocs.yml and remove this line

Reply:
If thats the case and we want to have it for all languages it should be in the `assets/landing` folder. If we want just english this is ok